force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -0,0 +1,112 @@
+import { LightningElement, api, track } from "lwc";
+import accountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+
+export default class EdaSettingsNavigation extends LightningElement {
+    labelReference = {
+        accountModelSettingsTitle,
+    };
+
+    @track settingsNavigationViewModel = {
+        navigationSections: [
+            {
+                label: "Setup Home",
+                page: "setuphome",
+                sectionkey: "setuphome",
+            },
+            {
+                label: "People and Groups",
+                sectionkey: "peopleandgroups",
+                navigationSubSections: [
+                    {
+                        label: this.labelReference.accountModelSettingsTitle,
+                        page: "accountmodelsettings",
+                        sectionkey: "accountmodelsettings",
+                        menuItems: [
+                            { label: "Default Account Model", sectionKey: "defaultaccountmodel" },
+                            {
+                                label: "Administrative Account Record Type",
```

Comment:
Same question regarding these labels...do they need to be translated?

Reply:
Likewise on translations. These are placeholders for now to be replaced by the 'real' labels.